### DISCUSSION TITLE
Test Summary isn't working on windows runner

### DISCUSSION THREAD

**Discussion by sgannap** [test-summary/action#23] [Q&A]
Here is my workflow file:
name: Node.js CI

on:
  push:
    branches:
      - '*'         # matches every branch that doesn't contain a '/'
      - '*/*'       # matches every branch containing a single '/'
      - '**'        # matches every branch
  pull_request:
    branches:
      - SmokeSuite
      - PreRelease
  schedule:
    - cron: "5 */4 * * *"

jobs:
  build:
    runs-on: "QA-Automation"

    strategy:
      matrix:
        node-version: [16]

    steps:
      - uses: actions/checkout@v3
      - name: Use Node.js ${{ matrix.node-version }}
        uses: actions/setup-node@v3
        with:
          node-version: ${{ matrix.node-version }}
      - run: npm run webtests
      - name: Setup Python
        uses: actions/setup-python@v2
        with:
          python-version: '3.7'
      - name: Publish Test Results
        uses: test-summary/action@v1
        if: always()
        with:
          paths: |
            "junit_reports/*.xml"
      - name: Test Report
        uses: dorny/test-reporter@v1
        if: success() || failure()    # run this step even if previous step failed
        with:
          name: Junit Tests            # Name of the check run which will be created
          path: "junit_reports/*.xml"   # Path to test results
          reporter: java-junit        # Format of test results 
      
      Test-reports are working as expected but not the summary part
<img width="609" alt="Screen Shot 2023-01-30 at 1 17 49 PM" src="https://user-images.githubusercontent.com/110626584/215562163-2c849544-f6fe-42b1-8348-746ad1a8f163.png">

     
<img width="287" alt="Screen Shot 2023-01-30 at 1 17 59 PM" src="https://user-images.githubusercontent.com/110626584/215562150-d4e7c260-8c58-4ffb-bef4-c2fd2b67a489.png">

**Comment by ethomson** [comment]
Apologies - this is working on my Windows runner. Can you provide some more details? Is this a public workflow?
